scripts/mainnet.sh

```diff
@@ -11,8 +11,8 @@ ARGS=(
         --genesis-hash            "5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb"
         --topology                "${TOPOLOGY}"
         --socket-dir              "./socket/"
-        --config                  "./configuration/mainnet.yaml"
+        --config                  "./configuration/configuration-mainnet.yaml"
         --port                    7776
 )
 
-${RUNNER} exe:cardano-node "${ARGS[@]}"
+${RUNNER} exe:cardano-node "${ARGS[@]}" $*
```

Comment:
:+1: Is this now part of all scripts that run the node?

Reply:
no just this one to pass in +RTS flags.